lib/python/ensembl/brc4/runnable/process_gff3.py

```diff
@@ -34,16 +34,10 @@ def param_defaults(self):
         return {
             "gene_types": (
                 "gene",
+                "pseudogene",
+                "transposable_element_gene",
+                "transposable_element_pseudogene",
```

Comment:
I thought we were not going to add these biotypes for now?

Reply:
Right I got ahead of myself there, I'll remove them